Source/UTags/Private/Tags.cpp

```diff
@@ -317,6 +317,69 @@ FString FTags::GetKeyValue(UObject* Object, const FString& TagType, const FStrin
 	return FString();
 }
 
+
+///////////////////////////////////////////////////////////////////////////
+// Get all the tags from the all the actors in the world which includes the Tags of each actor's components
+TMap<TWeakObjectPtr<UObject>, TArray<FTagData>> FTags::GetAllTagsFromWorldActors(UWorld * World)
+{ 
+	//Declaring our map data type that stores an object reference and all the tag related data in a defined struct
+	TMap<TWeakObjectPtr<UObject>, TArray<FTagData>> ActorsAndTagsMap;
+	//Iterate Actors from World
+	for (TActorIterator<AActor> ActorItr(World); ActorItr; ++ActorItr)
+	{
+		TWeakObjectPtr<AActor> WeakActorPtr = *ActorItr;
+		TArray<FName> ActorTags = WeakActorPtr->Tags;
+		TArray<FTagData> ActorTagsData;
+		if(WeakActorPtr.IsValid())
+		{
+			//Only add new tag data to valid weak pointer objects if they contain any tags
+			if (ActorTags.GetData() != nullptr) {
+			ActorTagsData = GetObjectTagsData(ActorTags, *ActorItr);
+			ActorsAndTagsMap.Add(WeakActorPtr, ActorTagsData);
+			}
+			//Iterate Components Of The Actor
+			for (const auto& CompItr : ActorItr->GetComponents())
+			{
+				TWeakObjectPtr<UActorComponent> WeakComponentPtr = CompItr;	
+				TArray<FName> ComponentTags = CompItr->ComponentTags;
+				TArray<FTagData> ComponentTagsData;
+
+				if (WeakComponentPtr.IsValid())
+				{
+					if (ComponentTags.GetData() != nullptr) {
+						ComponentTagsData = GetObjectTagsData(ComponentTags, CompItr);
```

Comment:
use namespace for the function calls e.g. FTags::GetObjectTagsData , it makes it clearer where the function is located